packages/chrome-extension/src/dashboard/dashboard.js

```diff
@@ -248,53 +124,81 @@ class DashboardRounds extends React.Component {
         super(props);
     }
 
+    renderPuzzle(puzzle) {
+        return r.a(
+            {
+                className: "Dashboard-puzzle " + puzzle.status,
+                href: "http://" + puzzle.host + puzzle.path
```

Comment:
```suggestion
                href: `http://${puzzle.host}${puzzle.path}`
```

we're fancy now like that